src/chart/components/pane/extent/y-extent-component.ts

```diff
@@ -46,8 +45,10 @@ export class YExtentComponent extends ChartBaseElement {
 		private hitTestController: PaneHitTestController,
 		public dataSeriesCanvasModel: CanvasModel,
 		public readonly scaleModel: ScaleModel,
-		// TODO when y-axis component will be refactored this shouldn't be undefined
-		public readonly yAxisComponent: ExtentYAxis | undefined,
+		newYAxisComponent: (
```

Comment:
If it is a function, name shold have an action (create, get and etc)

Reply:
makes sense